README.md

```diff
@@ -100,6 +100,15 @@ https://support.microsoft.com/en-gb/help/13862/windows-use-high-contrast-mode
 [how-users-change-colours-on-websites]:
 https://accessibility.blog.gov.uk/2017/03/27/how-users-change-colours-on-websites/
 
+## Getting updates
+
+To be notified when there’s a new release, you can either:
+
+- [watch the govuk-frontend Github repository](https://help.github.com/en/articles/watching-and-unwatching-repositories)
+- subscribe to the [#govuk-design-system channel on cross-government Slack](https://ukgovernmentdigital.slack.com/app_redirect?channel=govuk-design-system)
```

Comment:
Slack tends to talk about '[joining channels](https://get.slack.help/hc/en-gb/articles/205239967-Join-a-channel)' rather than 'subscribing to a channel', and I'm not aware of that phrase really being used.

Should we align with how Slack talks about these things, to avoid potential confusion? (for example, a user interpreting this to mean that you can subscribe to get channel updates by email)

Reply:
Make sense! I'll tweak.